<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-08T12:50:47 IST" name="Suite" finished-at="2025-08-08T12:51:09 IST" duration-ms="22107">
    <groups>
    </groups>
    <test started-at="2025-08-08T12:50:47 IST" name="Test" finished-at="2025-08-08T12:51:09 IST" duration-ms="22107">
      <class name="Test.MainTest">
        <test-method is-config="true" signature="initConfig()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:50:47 IST" name="initConfig" finished-at="2025-08-08T12:50:47 IST" duration-ms="378" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initConfig -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:50:47 IST" name="setUp" finished-at="2025-08-08T12:50:53 IST" duration-ms="5913" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="TestingFunctionalities()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:50:53 IST" name="TestingFunctionalities" finished-at="2025-08-08T12:50:54 IST" duration-ms="1149" status="FAIL">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[C:\Users\HP\eclipse-workspace\Wipro2\ScreenShots\Login Successful-1754637654785.png (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: C:\Users\HP\eclipse-workspace\Wipro2\ScreenShots\Login Successful-1754637654785.png (The system cannot find the path specified)
at java.base/java.io.FileOutputStream.open0(Native Method)
at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
at com.google.common.io.Files$FileByteSink.openStream(Files.java:197)
at com.google.common.io.Files$FileByteSink.openStream(Files.java:185)
at com.google.common.io.ByteSource.copyTo(ByteSource.java:279)
at com.google.common.io.Files.copy(Files.java:328)
at Test.MainTest.takeScreenshot(MainTest.java:169)
at Test.MainTest.TestingFunctionalities(MainTest.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestingFunctionalities -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:50:54 IST" name="tearDown" finished-at="2025-08-08T12:50:55 IST" duration-ms="492" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:50:55 IST" name="setUp" finished-at="2025-08-08T12:51:00 IST" duration-ms="4854" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testGetStatusCode()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:51:00 IST" name="testGetStatusCode" finished-at="2025-08-08T12:51:03 IST" duration-ms="2863" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetStatusCode -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:51:03 IST" name="tearDown" finished-at="2025-08-08T12:51:03 IST" duration-ms="383" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:51:03 IST" name="setUp" finished-at="2025-08-08T12:51:05 IST" duration-ms="2598" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testInvalidLogin()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:51:05 IST" name="testInvalidLogin" finished-at="2025-08-08T12:51:08 IST" duration-ms="2869" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.apache.poi.ss.usermodel.Sheet.getRow(int)" because "utils.ExcelReadUtils.sheet" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.poi.ss.usermodel.Sheet.getRow(int)" because "utils.ExcelReadUtils.sheet" is null
at utils.ExcelReadUtils.getCellValue(ExcelReadUtils.java:20)
at Test.MainTest.testInvalidLogin(MainTest.java:119)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInvalidLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:51:08 IST" name="tearDown" finished-at="2025-08-08T12:51:09 IST" duration-ms="540" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="tearDownSuite()[pri:0, instance:Test.MainTest@313ac989]" started-at="2025-08-08T12:51:09 IST" name="tearDownSuite" finished-at="2025-08-08T12:51:09 IST" duration-ms="421" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownSuite -->
      </class> <!-- Test.MainTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
